### ISSUE TITLE
VSCode doesn't connect to deno's debugger

### ISSUE THREAD

**Issue by AnInternetTroll** [denoland/vscode_deno#605]
When "Debug" is pressed above a `Deno.test` in [this file](https://git.aninternettroll.xyz/aninternettroll/arsoppgave.backend/src/commit/b14ff06429c1e7993b20873d8ceefb6b9443de08/tests/routes/auth.test.ts) the debugger will start but vscode doesn't connect

**To Reproduce**

1. Open [this project](https://git.aninternettroll.xyz/aninternettroll/arsoppgave.backend/src/commit/b14ff06429c1e7993b20873d8ceefb6b9443de08/) in vscode
2. Navigate to [tests/routes/auth.test.ts](https://git.aninternettroll.xyz/aninternettroll/arsoppgave.backend/src/commit/b14ff06429c1e7993b20873d8ceefb6b9443de08/tests/routes/auth.test.ts)
3. Press `Debug` above any `Deno.test`

**Expected behavior**

Debugging session starts and I can jump into and over functions and set breakpoints. 

**Screenshots**

![image](https://user-images.githubusercontent.com/40470127/148597914-395d4307-30fe-49da-a084-96f81ade222c.png)

**Versions**
Linux distro: Void Linux
VSCode was installed with [`sudo xbps-install -S vscode`](https://github.com/void-linux/void-packages/tree/master/srcpkgs/vscode)
```console
$ deno --version
deno 1.17.1+c40419b (canary, x86_64-unknown-linux-gnu)
v8 9.8.177.2
typescript 4.5.2
$ code-oss --version
Warning: 'app' is not in the list of known options, but still passed to Electron/Chromium.
1.63.0
Unknown commit
x64
$ uname -a
Linux wheatley 5.15.12_1 #1 SMP Wed Dec 29 19:18:57 UTC 2021 x86_64 GNU/Linux
```

**Comment by kitsonk** [comment]
What version of the vscode_deno extension is installed?

**Comment by AnInternetTroll** [comment]
v3.10.0

**Comment by clemens-tolboom** [comment]
I can set a breakpoint in my test then run the test (click play once on first code line it seems) next breakpoint paused.

Even tested on deno_std git checkout running one of  `fs/move_test.ts`

![image](https://user-images.githubusercontent.com/371014/148951463-42edfbfd-ea91-467a-bf64-819cd135805b.png)

```
deno --version
deno 1.17.1 (release, x86_64-apple-darwin)
v8 9.7.106.15
typescript 4.5.2
```

vscode_deno: v3.10.0

## VSCode
```
Version: 1.63.2
Commit: 899d46d82c4c95423fb7e10e68eba52050e30ba3
Date: 2021-12-15T09:37:28.172Z (3 wks ago)
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin x64 20.6.0
```

**Comment by Leokuma** [comment]
This may have been fixed in v3.16.0
